README.md

```diff
@@ -1,32 +1,34 @@
+1. `MinimalWordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/MinimalWordCount.java), [python](python/dataflow_examples/wordcount_minimal.py)) is the simplest word count pipeline and introduces basic concepts like [Pipelines](https://cloud.google.com/dataflow/model/pipelines),
 [PCollections](https://cloud.google.com/dataflow/model/pcollection),
 [ParDo](https://cloud.google.com/dataflow/model/par-do),
 and [reading and writing data](https://cloud.google.com/dataflow/model/reading-and-writing-data) from external storage.
 
-1. [`WordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/WordCount.java) introduces Dataflow best practices like [PipelineOptions](https://cloud.google.com/dataflow/pipelines/constructing-your-pipeline#Creating) and custom [PTransforms](https://cloud.google.com/dataflow/model/composite-transforms).
+1. `WordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/WordCount.java), [python](python/dataflow_examples/wordcount.py)) introduces Dataflow best practices like [PipelineOptions](https://cloud.google.com/dataflow/pipelines/constructing-your-pipeline#Creating) and custom [PTransforms](https://cloud.google.com/dataflow/model/composite-transforms).
 
-1. [`DebuggingWordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/DebuggingWordCount.java)
+1. `DebuggingWordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/DebuggingWordCount.java), [python](python/dataflow_examples/wordcount_debugging.py))
 shows how to view live aggregators in the [Dataflow Monitoring Interface](https://cloud.google.com/dataflow/pipelines/dataflow-monitoring-intf), get the most out of
 [Cloud Logging](https://cloud.google.com/dataflow/pipelines/logging) integration, and start writing
 [good tests](https://cloud.google.com/dataflow/pipelines/testing-your-pipeline).
 
-1. [`WindowedWordCount`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples/WindowedWordCount.java) shows how to run the same pipeline over either unbounded PCollections in streaming mode or bounded PCollections in batch mode.
+1. `WindowedWordCount`([java](java/examples/src/main/java/com/google/cloud/dataflow/examples/WindowedWordCount.java)) shows how to run the same pipeline over either unbounded PCollections in streaming mode or bounded PCollections in batch mode.
 
 ## Building and Running
 
-The examples in this repository can be built and executed from the root directory by running:
+### Java
+
+The examples in this repository can be built and executed from the `java` directory by running:
```

Comment:
repository is no longer accurate. example module?

java directory isn't accurate either. you have to be in a subfolder of java.